src/bin/wasmldr/src/loader/configured/mod.rs

```diff
@@ -50,37 +53,21 @@ impl Loader<Configured> {
     }
 
     pub fn next(self) -> Result<Loader<Requested>> {
-        let platform = Platform::get()?;
-        let cert_algo = match platform.technology() {
-            Technology::Snp => SECP_384_R_1,
-            Technology::Sgx => SECP_256_R_1,
-            Technology::Kvm => SECP_256_R_1,
-        };
+        // TODO:
+        //   1. get attestation
+        //   2. get intermediate cert
+        //   3. choose key type based on technology
+        //   4. generate tech-specific attestation extension request
 
         // Generate a keypair.
-        let raw = PrivateKeyInfo::generate(cert_algo)?;
+        let raw = PrivateKeyInfo::generate(SECP_256_R_1)?;
         let pki = PrivateKeyInfo::from_der(raw.as_ref())?;
-        let der = pki.public_key().unwrap().to_vec().unwrap();
-
-        let mut key_hash = [0u8; 64];
-        match platform.technology() {
-            Technology::Snp => {
-                let hash = Sha384::digest(der);
-                key_hash[..48].copy_from_slice(&hash);
-            }
-            _ => {
-                let hash = Sha256::digest(der);
-                key_hash[..32].copy_from_slice(&hash);
-            }
-        };
-
-        let attestation_report = platform.attest(&key_hash)?;
```

Comment:
objections!

Reply:
This commit is rejected.

Reply:
AKA it is not really part of the PR but I need to have there while developing.